cfn-resources/util/validator/validator.go

```diff
@@ -18,22 +17,7 @@ type ValidatorDefinition interface {
 	GetListFields() []string
 }
```

Comment:
Is this interface still in use? Can it be removed?

Reply:
Yes.We have removed this interface